docs-src/modules/getting-started/pages/index.adoc

```diff
@@ -1,14 +1,91 @@
 = Getting Started
 
-Kaskada's query language xref:overview:what-is-kaskada.adoc[focuses on event-based data processing] making it easier to reason about when things happen, the state at specific points in time, and how results change over time.
-Kaskada queries are simple, declarative and composable providing a more expressive way to query event-based data.
+== Reasoning about events with timelines
 
-There are several ways to quickly get started with Kaskada.
+Kaskada is a unified event processing engine that provides stateful stream processing in a high-level query language designed specifically for reasoning about events in bulk and in real time.
+Kaskada is built on the idea of a _timeline_ - the history of how a value changes over time for a specific entity or group.
 
-* xref:getting-started:hello-world-jupyter.adoc[]: How to get up and running with Kaskada inside a Jupyter notebook running locally. 
-This quick tutorial will help you install and run Kaskada from within a Jupyter notebook, load some sample data into Kaskada, use Kaskada queries and Python within Jupyter to explore that data.
-* xref:getting-started:hello-world-cli.adoc[]: How to execute queries from the command line using the Kaskada CLI. 
-This quick tutorial will help you install and run Kaskada from the command line, load some sample data into Kaskada, and use Kaskada queries to explore that data all on the command line.
+[stream_viz,name=basic-sum]
+.Aggregating events as a timeline
+....
+[
+    {
+        "label": "Purchase.amount",
+        "kind": "discrete",
+        "data": [
+            [
+                {"t": 2, "v": "5"},
+                {"t": 4, "v": "2"},
+                {"t": 13, "v": "13"},
+                {"t": 18, "v": "4"}
+            ]
+        ]
+    },
+    {
+        "label": "Purchase.amount | sum()",
+        "kind": "continuous",
+        "data": [
+            [
+                {"t": 2, "v": "5"},
+                {"t": 4, "v": "7"},
+                {"t": 13, "v": "20"},
+                {"t": 18, "v": "24"}
+            ]
+        ]
+    }
+]
+....
+
+Timelines give you declarative queries and the ability to reason about temporal context, time travel, sequencing, timeseries, and more.
+Transforming and combining timelines allow you to intuitively express computations over events.
+
+****
+⭢ Read more about timelines in xref:overview:what-is-kaskada.adoc[]
+****
+
+With Kaskada, it's timelines all the way down - every operation starts from timelines and produces timelines. 
```

Comment:
Terminology "converted to a table" may make people think this doesn't produce streams. Maybe:

When you make a query, the resulting timeline is converted to data in one of two ways -- as a timeline history or as a timeline snapshot.

* The *timeline history* generates a row for every point the timeline changed, and each row is associated with the time it changed and the corresponding entity.
* The *timeline snapshot* generates a row for each entity at the _same_ point in time. Each row is associated with a different entity, but all rows are associated with the same time.

These rows may be output to a stream or table store.
See ... for information on outputting timelines.